src/logic/translations.js

```diff
@@ -336,6 +337,37 @@ const texts = {
       '333mbf': '3x3x3 Multi-Blind',
     },
   },
+  'pt-br': {
+    eventLabel: 'Categoria',
+    round: 'Rodada',
+    group: 'Grupo',
+    station: 'Est',
+    name: 'Nome',
+    newCompetitor: 'Novo competidor',
+    scr: 'Emb',
+    check: 'Confere',
+    result: 'Resultado',
+    judge: 'Juiz',
+    extra: 'Tentativa extra',
+    delegateInitials: 'Iniciais do delegado',
+    cutoff: 'Tempo de corte',
+    timeLimit: 'Tempo limite',
+    total: 'cumulativo',
+    eventName: {
+      '333': 'Cubo 3x3x3',
+      '222': 'Cubo 2x2x2',
+      '444': 'Cubo 4x4x4',
+      '555': 'Cubo 5x5x5',
+      '666': 'Cubo 6x6x6',
+      '777': 'Cubo 7x7x7',
+      '333bf': '3x3x3 Vendado',
+      '333fm': '3x3x3 em Menos Movimentos',
+      '333oh': '3x3x3 com Uma Mão',
```

Comment:
It's the same, but it makes sense to put these, so it's clear nothing is missing :)

```suggestion
      '333oh': '3x3x3 com Uma Mão',
      minx: 'Megaminx',
      pyram: 'Pyraminx',
      clock: 'Clock',
      skewb: 'Skewb',
      sq1: 'Square-1',
```